lesson2/summ.py

```diff
@@ -0,0 +1,7 @@
+if __name__ == "__main__":
+    x = 0
+    str1 = input('insert the numbers: ')
+    lst1 = str1.split()
+    for i in range(len(lst1)):
```

Comment:
This is c++ for cycle style. Use python style instead
```suggestion
    for i in lst1:
```